recipe/meta.yaml

```diff
@@ -12,42 +12,47 @@ source:
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record=files.txt
-  skip: true  # [win or py3k]
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: true  # [win]
 
 requirements:
   build:
-    - aipy >=2.1.6
-    - astropy >=1.2
-    - numpy >=1.11
-    - pyephem
-    - python <3
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - six >=1.10
+    - astropy >=2.0
+    # we require at least numpy 1.14 in pyuvdata
+    - numpy >=1.14
+    - pip
+    - python
     - scipy
     - setuptools
   run:
-    - aipy >=2.1.6
-    - astropy >=1.2
-    - numpy >=1.11
-    - pyephem
-    - python <3
+    - six >=1.10
+    - astropy >=2.0
+    # we require at least numpy 1.14 in pyuvdata
+    - numpy >=1.14
```

Comment:
Change this to `- {{ pin_compatible('numpy') }}`

Reply:
Thanks @isuruf.